src/extension.ts

```diff
@@ -725,7 +727,7 @@ export function activate(context: vscode.ExtensionContext) {
               context.subscriptions
             );
           }
-        } catch(err) {
+        } catch (err) {
```

Comment:
print the error please

Reply:
Here also I did print the error message in next line.